infra.yaml

```diff
@@ -8,163 +8,252 @@ description: >
+    description: >
+      The SSH public key that Ansible will use to access master and node hosts
+      This will be placed on each VM host in /root/.ssh/authorized_keys
     type: string
 
   ansible_private_key:
+    description: >
+      The private key that Ansible will use to access master and node hosts
+      This file will be placed on the infrastructure host and protected
     type: string
 
+  # OpenShift service parameters
   deployment_type:
+    description: >
+      Select the community or enterprise version of OpenShift
     type: string
 
   lb_hostname:
+    description: >
+      The hostname of the load balancer feeding the OpenShift master traffic
     type: string
     default: ''
 
   ldap_url:
+    description: >
+      The LDAP url for access to the LDAP service (redundant? ML)
     type: string
     default: ''
 
   ldap_preferred_username:
+    description: >
+      What field to use to look up and identify users in the database
     type: string
     default: 'uid'
 
   ldap_bind_dn:
+    description: >
+      LDAP service access user identifier
     type: string
     default: ''
 
   ldap_bind_password:
+    description: >
+      LDAP service access password
     type: string
     default: ''
 
   ldap_ca:
+    description: >
+      The Certificate Authority file to confirm SSL encryption security
     type: string
     default: ''
 
   ldap_insecure:
+    description: >
+      Toggle SSL encryption for LDAP communications
     type: string
     default: false
 
   openshift_sdn:
+    description: >
+      Enable and select the container communications network: OVS or Flannel
     type: string
 
   dns_ip:
+    description: >
+      A list of external DNS servers to resolve external hosts
     type: string
-    description: IP address of the DNS server
 
   docker_volume_size:
-    type: number
     description: >
-      size of a cinder volume in GB to allocate to docker for container/image
-      storage
+      The size of a cinder volume in GB to allocate to docker for
+      container/image storage
+    type: number
     default: 25
 
   floating_ip:
+    description: >
+      TBD: What is this a floating IP for? For the DNS server?
     type: string
 
   port:
     description: Neutron port (with a floating IP address) to assign to the DNS Nova Server
     type: string
 
+  # Load Balancer
   lb_ip:
+    description: >
+      The IP address of the load balancer feeding the OpenShift master traffic
     type: string
     default: ''
 
+  # LDAP
+  # For external and/or common user information with OpenStack
   ldap_ip:
+    description: >
+      The IP address of the LDAP server providing OpenShift user information
     type: string
     default: ''
 
   ldap_hostname:
+    description: >
+      The hostname of the LDAP server providing OpenShift user information
     type: string
     default: ''
 
   deploy_registry:
+    description: >
+      Enable or disable the internal Docker registry
```

Comment:
I think it's more "Enable or disable automatic provisioning of the Docker registry"